### PR TITLE
Signaler QoL

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=30&p=247728#p247542

THIS KILLS TACTICAL SIGNALERS

🆑 Cobby
add: You can now copy one signaler's settings to another by hitting it on the other signaler. [Credit to Weepo for the idea]
/🆑

> 2557 > using github application 
> 2557 > testing your code

if someone could make that changelog simpler please hmu

In case someone was confused on the function.
* You have Signaler1 and Signaler2
* Signaler 1 is in active hand
* clicking signaler2 with hand+signaler1 edits **Signaler2**'s code and frequency

### CODE REVIEW

**Comment by Core0verload** [comment]
\> 2557 > not knowing how to use markdown

**Comment by Kromgar** [comment]
POWERCREEP

**Comment by Supermichael777** [comment]
Its add:
not :add:

**Comment by ExcessiveUseOfCobblestone** [comment]
thank you

**Comment by KorPhaeron** [comment]
Should add the functionality to a signalers description  so people have a way of knowing this exists

**Comment by Cyberboss** [comment]
>#23289 gets labelled as a tweak
>This gets labelled as a feature

C O N S I S T E N C Y

File: code/modules/assembly/signaler.dm [hunk 1]
```diff
@@ -96,7 +96,16 @@ Code:
...
 	return
 
-
+/obj/item/device/assembly/signaler/attackby(obj/item/weapon/W, mob/user, params)
+	if(issignaler(W))
+		var/obj/item/device/assembly/signaler/signaler2 = W
+		if(secured && signaler2.secured)
+			code = signaler2.code
+			frequency = signaler2.frequency
+			user << "You transfer the frequency and code of \the [signaler2.name] to \the [name]"
```
Comment (Fox-McCloud) [inline code/modules/assembly/signaler.dm]: `user << "You transfer the frequency and code of [signaler2.name] to [src]"`

Comment (ExcessiveUseOfCobblestone) [inline code/modules/assembly/signaler.dm]: stinky src user

**Pull request merged**